graylog2-web-interface/src/components/users/PermissionSelector.test.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Immutable from 'immutable';
+import { mount } from 'enzyme';
+
+import PermissionSelector from 'components/users/PermissionSelector';
+
+/* https://github.com/facebook/react/issues/7371
+ *
+ * findDomNode with refs is not supported by the react-test-renderer.
+ * So we need to mock the findDOMNode function for TableList respectievly
+ * for its child component TypeAheadDataFilter.
+ */
+jest.mock('react-dom', () => ({
```

Comment:
As we may need this function in other tests, maybe we can extract it into a mocking helper in `graylog2-web-interface/test/helpers/mocking`. We can also ignore it for now, but I guess it's likely that we end up copy-pasting this code here and there :)